operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md

```diff
@@ -238,39 +238,42 @@ Before redeploying MEDS, update the `customizations.yaml` file in the `site-init
-            ```
+    1. Verify that the BMC xname is resolvable and pingable:
     
-            If the slot is off, power it on:
-            ```bash
-            ncn-m001# cray capmc xname_on create --xnames x1001c1s0
-            ```
- 
-        3. If the BMC is reachable and in `HTTPsGetFailed`, then verify that the BMC is accessible with the new default global credential. Replace `BMC_HOSTNAME` with the hostname of the Redfish Endpoint. For a NodeBMC its hostname is its xname. For a ChassisBMC, then the `b0` in its xname needs to be removed to get its hostname.
-            > For example, the ChassisBMC has the xname `x1000c0b0`, and its hostname is `x1000c0`.
-            > <!-- In CSM 1.0 the hostname for a ChassisBMC matches its xname -->
-            ```bash
-            ncn-m001# curl -k -u root:$CRED_PASSWORD https://BMC_HOSTNAME/redfish/v1/Managers | jq
-            ```
-
-            If the error message below is returned, then the BMC needs to be have a StatefulReset action performed on it. The StatefulReset action will clear out any previously user defined credentials that are taking precedence over the CEC supplied credential. It will also clear out NTP, Syslog, and SSH Key configurations on the BMC.
+       > If the BMC is a ChassisBMC, then the `b0` in its xname needs to be removed to get its hostname. Otherwise, for NodeBMCs their xnames is their BMC hostname. For example, the ChassisBMC has the xname `x1000c0b0`, and its hostname is `x1000c0`.
+    
+       ```bash
+        ncn-m001# ping x1001c1s0b0
+       ```
+    
+    2. If a NodeBMC is not pingable, then verify that the slot powering the BMC is powered on. If this is a ChassisBMC, skip this step. For example, the NodeBMC x1001c1s0b0 is in slot x1001c1s0.
+    
+       ```bash
+        ncn-m001# cray capmc get_xname_status create --xnames x1001c1s0
+               e = 0
+               err_msg = ""
+               on = [ "x1001c1s0b0",]
+       ```
+    
+       If the slot is off, power it on:
+    
+       ```
+        ncn-m001# cray capmc xname_on create --xnames x1001c1s0
+       ```
+    
+    3. If the BMC is reachable and in `HTTPsGetFailed`, then verify that the BMC is accessible with the new default global credential. Replace `BMC_HOSTNAME` with the hostname of the Redfish Endpoint. For a NodeBMC its hostname is its xname. For a ChassisBMC, the `b0` part of the xname must be removed to get its hostname.
+    
+       > For example, the ChassisBMC has the xname `x1000c0b0`, and its hostname is `x1000c0`.
     
+       ```bash
+       ncn-m001# curl -k -u root:$CRED_PASSWORD https://BMC_HOSTNAME/redfish/v1/Managers | jq
+       ```
+    
+        If the error message below is returned, then the BMC requires a StatefulReset action. The StatefulReset action will clear user-defined credentials that are taking precedence over the CEC supplied credential. It will also clear NTP, Syslog, and SSH Key configurations on the BMC.
+
             ```json
```

Comment:
This part of the page isn't quite rendering right in Github. I think it is due to all of the sub-sub-steps getting moved 1 level to the left, and this didn't move is causing the issue.
<img width="1345" alt="Screen Shot 2021-10-21 at 3 07 49 PM" src="https://user-images.githubusercontent.com/77114568/138349528-2f740815-e46f-4adb-905b-62c620ecb9f3.png">